### DISCUSSION TITLE
Doc comment on Collection schema (/config-refs/schema)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1265] [Docs/blog comments]
# Doc comment on Collection schema (/config-refs/schema)

Introduction

https://weaviate.io/developers/weaviate/config-refs/schema

**Comment by hughesadam87** [comment]
This article references searching and filtering multiple times, but the difference is not clear to me.  For example, what is the layment difference in these:

    indexSearchable
    indexFilterable

Isn't filtering done through the search operator's `with_where()` method?

**Comment by sirish-amatya** [comment]
Is there a way to implement custom tokenization method? Eg. I want to use a tokenization method that uses porter stem instead of word and also have a synonym list during tokenization so that IT could give me results for Information Technology. Elasticsearch provides a way to tokenize using standard method and allows us to apply filters for porter stem and synonym. Do we have anything like that in Weviate?
